### PR TITLE
fix iFARM; migrate to ENS

### PR DESCRIPTION
**Pull request by brandoncurtis**
#### Hi! What is your PR about?

- [ ] Add images for your space
- [ ] Add custom domain for your space
- [x] Add alias for migrate your space
- [ ] Add or edit a skin
- [x] Edit your space with GitHub? It's recommended to migrate to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: spaces/farm/aliases.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "farm": "harvestfi.eth"
+}
```
Comment (gokaiorg) [inline spaces/farm/aliases.json:3]: You don't have to create an aliases file in your space directory, you need to add your alias in the list of this file https://github.com/snapshot-labs/snapshot-spaces/blob/master/spaces/aliases.json

**Review by gokaiorg — CHANGES_REQUESTED** [review]
- I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.page/#/harvestfi.eth
- Your space name directory should be similar to your ENS. From `spaces/farm/logo.png` to `spaces/harvestfi.eth/logo.png`.
- Since we use ENS and IFPS, every datas are sets in your space settings, you shouldn’t add index.json file. Please remove it.

**Comment by brandoncurtis** [comment]
Got it, so you migrate current votes but you don't import any of the other settings—we have to do that manually in the web UI.

**Comment by gokaiorg** [comment]
> Got it, so you migrate current votes but you don't import any of the other settings—we have to do that manually in the web UI.

Yes, you have to redo your configurations in the parameters of your space so that you are free to modify them without going through PR.

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**